internal/storage/sqlindexer_test.go

```diff
@@ -32,7 +32,7 @@ func TestSQLInit(t *testing.T) {
 	options, err := CreateFakeIndexerOptions(db, swapDb)
 	require.NoError(t, err)
 
-	fs := PrepareFakeServer(t, "testdata/search-index-all-full.json")
+	fs := PrepareFakeServer(t, "../../storage/testdata/search-index-all-full.json")
```

Comment:
Changed all tests to use the files from `storage/testdata` folder.
They were the same files, and it was needed to update both at the same time. Let's keep just one copy.